resource/order.go

```diff
@@ -1,26 +1,26 @@
-package endpoint
+package resource
 
 import (
 	"encoding/json"
+	"errors"
 	"fmt"
 	"github.com/GFG/seller-center-sdk-go/client"
 	"github.com/GFG/seller-center-sdk-go/model"
 	"github.com/buger/jsonparser"
-	"github.com/pkg/errors"
 	"strconv"
 	"strings"
 	"time"
 )
 
-type OrderEndpoint struct {
+type OrderResource struct {
 	client client.Client
 }
 
-func NewOrder(client client.Client) OrderEndpoint {
-	return OrderEndpoint{client: client}
+func NewOrder(client client.Client) OrderResource {
+	return OrderResource{client: client}
 }
 
-func (oe OrderEndpoint) GetOrders(
+func (oe OrderResource) GetOrders(
```

Comment:
please change the variable to "or" instead of "oe" i think oe is for OrderEndpoing